tests/trainer/test_sharded_checkpoint.py

```diff
@@ -422,3 +424,41 @@ def test_sharded_folder(world_size, use_remote, tmp_path: pathlib.Path, state_di
     )
     trainer2.fit()
     trainer2.close()
+
+@pytest.mark.gpu
+@world_size(2)
+@pytest.mark.parametrize('state_dict_type', ['local', 'sharded'])
+@pytest.mark.parametrize('weights_only', [True, False])
+@pytest.mark.skipif(version.parse(torch.__version__) < version.parse('2.0.0'),
+                    reason='requires PyTorch 2.0.0 or higher')
+@pytest.mark.filterwarnings(r'ignore:TypedStorage is deprecated.:UserWarning')
+def test_new_sharded_save(world_size, tmp_path: pathlib.Path, state_dict_type: str, weights_only: bool):
```

Comment:
more descriptive name than "new" please

Reply:
it's just a temporary test until we get load working